### PR TITLE
Tcec hvac

### PR DESCRIPTION
**Pull request by k8mckay**
TCEC THEC for HVAC1 and HVAC2: all tests passing

### CODE REVIEW

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC2/certificate_estimation/HVAC2_ESC_calculation.py [hunk 1]
```diff
@@ -93,7 +93,27 @@ def formula(buildings, period, parameters):
                         (heating_capacity * equivalent_heating_hours) / baseline_ACOP
                     ])
 
-  
+class HVAC2_THEC(Variable):
```
Comment (alisha17) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC2/certificate_estimation/HVAC2_ESC_calculation.py:96]: can we change the names of these variables to something more informative
like HVAC2_thec_or_equation

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC1/certificate_estimation/HVAC1_ESC_calculation.py [hunk 2]
```diff
@@ -95,7 +95,28 @@ def formula(buildings, period, parameters):
                         (heating_capacity * equivalent_heating_hours) / baseline_ACOP
                     ])
 
+
+class HVAC1_THEC(Variable):
```
Comment (alisha17) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC1/certificate_estimation/HVAC1_ESC_calculation.py:99]: can we change this variable name to something more informative?

like HVAC1_thec_or_annual_reference_heating

Comment (k8mckay) [reply openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC1/certificate_estimation/HVAC1_ESC_calculation.py:99]: Okay will do - hold the line

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC2/certificate_estimation/HVAC2_ESC_calculation.py (hunk 1 above)
Comment (k8mckay) [reply openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC2/certificate_estimation/HVAC2_ESC_calculation.py:96]: Okay will do - hold the line

**Review by alisha17 — APPROVED** [review]

**Pull request merged**